### DISCUSSION TITLE
Compilation error

### DISCUSSION THREAD

**Discussion by pf-and-more** [kachurovskiy/nanoels#205] [General]
Hello! I read about this wonderful project some time ago and I decided to proceed with my own implementation! I had my PCBAs assembled by JLCPCB (modified versions on 2 layers PCBs, but fundamentally the same project) and I just received them today. Yay!

The problem is I get this compilation error in Arduino IDE 2.3.2:

![comp_err](https://github.com/user-attachments/assets/45a687e7-5684-443c-bc0d-5559c2ad03f9)

My target board is set as "ESP32S3 Dev Module". Anyone else is getting this compilation error?
Thank you all!

pf

**Comment by kachurovskiy** [comment]
Hi @pf-and-more, congratulations! Here's how to fix the error - https://github.com/kachurovskiy/nanoels/issues/199

Please share your PCB files if you can, thanks!

**Reply by pf-and-more** [reply]
Thank you Maxim! It compiles now.
My layout is attached here, I hope someone can find it useful :)

[2L-Nano-ELS.zip](https://github.com/user-attachments/files/16197820/2L-Nano-ELS.zip)

**Reply by Kosnov** [reply]
Hello.
Don't install any updates!
Remove Arduino and all its folders.
ESP32 - 2.0.8
TSA8418 - 1.0.0
